sdk/boms/azure-spring-cloud-dependencies/pom.xml

```diff
@@ -49,7 +49,7 @@
     <azure.identity.version>1.3.5</azure.identity.version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
     <azure.servicebus.version>7.4.0</azure.servicebus.version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;dependency} -->
     <azure.resourcemanager.version>2.7.0</azure.resourcemanager.version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager;dependency} -->
-    <azure.spring.appconfigure.version>2.0.0</azure.spring.appconfigure.version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-appconfiguration-config-web;dependency} -->
+    <azure.spring.appconfigure.version>2.1.0</azure.spring.appconfigure.version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-appconfiguration-config-web;current} -->
```

Comment:
I think we should the update version tag also, otherwise, it could cause problems when bumping the version after the release.